barman/utils.py

```diff
@@ -902,3 +902,39 @@ def lock_files_cleanup(lock_dir, lock_directory_cleanup):
                 _logger.debug(
                     "%s file lock already acquired, skipping removal" % filename
                 )
+
+
+def edit_config(file, section, option, value):
```

Comment:
I'd say we could change this method to receive a dict or a list of tuples, so we can update several options in a batch, and only after that call the open function and write method.